Dockerfile

```diff
@@ -1,15 +1,12 @@
-ARG PYTHON_VERSION=3.8.6-slim-buster
-
-FROM python:$PYTHON_VERSION
-
-ARG MLFLOW_VERSION=1.11.0
+FROM python:3.8.10-slim-buster
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-        libyaml-cpp-dev=0.6.2-4 libyaml-dev=0.2.1-1 && \
+    libyaml-cpp-dev=0.6.2-4 libyaml-dev=0.2.1-1 default-libmysqlclient-dev=1.0.5 gcc=4:8.3.0-1 && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
-RUN pip install psycopg2-binary==2.8.6 mlflow[extras]==$MLFLOW_VERSION
+COPY requirements.txt requirements.txt
```

Comment:
I would do `COPY requirements.txt /requirements.txt` to make destination path more explicit

Reply:
good idea, moved this file in /tmp/requiements.txt and removed it after use in  5469603